core/image/image.proto

```diff
@@ -93,25 +86,30 @@ message FmtATC_RGBA_EXPLICIT_ALPHA_AMD {
 }
 message FmtATC_RGBA_INTERPOLATED_ALPHA_AMD {
 }
+
 message FmtETC1_RGB_U8_NORM {
 }
-message FmtETC2_RGB_U8_NORM {
-  bool srgb = 1;
-}
-message FmtETC2_RGBA_U8_NORM {
-  bool srgb = 1;
-}
-message FmtETC2_RGBA_U8U8U8U1_NORM {
-  bool srgb = 1;
-}
-message FmtETC2_R_U11_NORM {
-}
-message FmtETC2_RG_U11_NORM {
-}
-message FmtETC2_R_S11_NORM {
-}
-message FmtETC2_RG_S11_NORM {
+
+message FmtETC2 {
+  enum ColorMode {
+    R = 0;
+    RG = 1;
+    RGB = 2;
+    RGBA = 3;
```

Comment:
RGBA is redundant. RGBA is RGB + alpha != ALPHA_NONE.
Also, signed = true makes no sense for most formats, similarly srgb = true. Maybe instead of having these extra booleans, you should expand the ColorMode enum to be : R, SIGNED_R, RG, SIGNED_RG, RGB and SRGB. If we did that, the only non-sense would be an alpha mode of anything but NONE for R, SIGNED_R, RG, and SIGNED_RG, but I think I could live with that.